js/main.js

```diff
@@ -1,6 +1,33 @@
-import { getPictures } from './data.js';
+//import { getPictures } from './data.js';
 import { renderGallery } from './gallery.js';
-import { setupForm } from './form.js';
+//import { setupForm, setUserFormSubmit } from './form.js';
+//import { showAlert } from './util.js';
+import { getData} from './api.js';
+const data = getData();
+renderGallery(data);
```

Comment:
getData возвращает промис, так что нужен либо  then либо await
```suggestion
const data = await getData();
renderGallery(data);
```